src/test/java/org/jenkinsci/plugins/oic/PluginTest.java

```diff
@@ -822,6 +846,14 @@ private static void assertTestUserEmail(User user) {
                 "Email should be " + TEST_USER_EMAIL_ADDRESS);
     }
 
+    private static void assertTestAvatar(User user, WireMockExtension wireMock) {
+        String expectedAvatarUrl = "http://localhost:%d/my-avatar.png".formatted(wireMock.getPort());
```

Comment:
use `WiremockRule.baseUrl()` or WireMockRule.url(String)?

(untested)
```suggestion
        String expectedAvatarUrl = WiremockRule.url("my-avatar.png");
```